File: FestivalJava/src/main/java/Utils/Observable.java
package Utils;

/**
 * Created by Sebi on 19-Mar-17.
 */
public interface Observable {
    /*
    Adds an observer
     */
    void addObserver(Observer o);

    /*
    Removes an observer
     */
    void removeObserver(Observer o);

    /*
    Notify observers
     */
    void notifyObservers();
}

File: FestivalC#/Festival/Utils/Observable.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace Utils
{
    public interface Observable<E>
    {
        void addObserver(Observer<E> o);
        void removeObserver(Observer<E> o);
        void notifyPushObservers(E e);
    }
}
